lib/fast_jsonapi/object_serializer.rb

```diff
@@ -1,36 +1,45 @@
 require 'active_support/core_ext/object'
 require 'active_support/concern'
 require 'active_support/inflector'
+require 'active_support/notifications'
 require 'oj'
 require 'multi_json'
 require 'fast_jsonapi/serialization_core'
 
-begin
-  require 'skylight'
-  SKYLIGHT_ENABLED = true
-rescue LoadError
-  SKYLIGHT_ENABLED = false
-end
-
 module FastJsonapi
   module ObjectSerializer
+
+    if ENV['SERIALIZATION_INSTRUMENTATION_ENABLED'.freeze] == 'true'.freeze
+      INSTRUMENTATION_ENABLED = true
+    elsif ENV['SERIALIZATION_INSTRUMENTATION_DISABLED'.freeze] == 'true'.freeze
+      INSTRUMENTATION_ENABLED = false
+    else
+      INSTRUMENTATION_ENABLED = false
+    end
+
+    SERIALIZE_HASH_NOTIFICATION = 'render.fast_jsonapi.serializable_hash'.freeze
+    TO_JSON_HASH_NOTIFICATION = 'render.fast_jsonapi.to_json'.freeze
+
     extend ActiveSupport::Concern
     include SerializationCore
 
     included do
-      # Skylight integration
-      # To remove Skylight
-      # Remove the included do block
-      # Remove the Gemfile entry
-      if SKYLIGHT_ENABLED
-        include Skylight::Helpers
-
-        instrument_method :serializable_hash
-        instrument_method :to_json
-      end
-
       # Set record_type based on the name of the serializer class
       set_type default_record_type if default_record_type
+
+      self.class_eval do
+        alias_method :to_json_without_instrumentation, :to_json
+
+        def to_json
```

Comment:
we dont have a method called `to_json`. I think the method we use is called `serialized_json` to get the JSON string representation. 

I like the simple and readable approach you have taken with `serializable_hash` can we do the same with `serialized_json`?

Reply:
I was going off the original Skylight instrumenting, [here](https://github.com/Netflix/fast_jsonapi/pull/61/files#diff-34088120ad9556cc6578b0c51b38c723L29), which did `serializable_hash` and `to_json`. It would be easy enough to change.